### PR TITLE
Change bash to sh

### PR DESCRIPTION
**Pull request by drozdowsky**
I want to merge this change because...
there is no need to use bash
Thank you

### Screenshots
none

### Pull Request Checklist
1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] Database migration files are up to date.
1. [x] The changes are tested.
1. [ ] GraphQL schema and type definitions are up to date.
1. [x] Changes are mentioned in the changelog.

### CODE REVIEW

**Comment by django-queries** [comment]
Here is the report for ac30f339dd072e2138887ee82fbef3656373dba8 (drozdowsky/saleor @ master)
Base comparison is 25f8ff78d3175adf61bcfbea96851b9b774b1eb9.



<details><summary>No differences were found. (click me)</summary>
<p>

```diff
# api.benchmark checkout
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  add billing address to checkout            	         34	         34	             20
  add shipping to checkout                   	          7	          7	              0
  checkout payment charge                    	         10	         10	              0
  complete checkout                          	          8	          8	              0
  create checkout                            	         50	         50	             24

# api.benchmark homepage
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  retrieve main menu                         	          5	          5	              0
  retrieve product list                      	          4	          4	              0
  retrieve secondary menu                    	          5	          5	              0
  retrieve shop                              	          2	          2	              0

# api.benchmark product
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product details                            	         15	         15	              3
  retrieve product attributes                	          9	          9	              0

# api.benchmark variant
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  product variant bulk create                	         51	         51	              3
  retrieve variant list                      	         15	         15	              6

# api product sorting attributes
  test name                                  	left count 	right count	duplicate count
  -------------------------------------------	-----------	-----------	---------------
  sort product not having attribute data     	         21	         21	              0
```

</p>
</details>

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -40,7 +40,7 @@ repos:
       - id: migrations-check
         language: system
         name: Check for uncreated migrations.
-        entry: bash -c "./manage.py makemigrations --check --dry-run"
+        entry: sh -c "./manage.py makemigrations --check --dry-run"
```
Comment (slopeofhope81) [inline .pre-commit-config.yaml]: hey~ just a question. what is the advantage of using `sh` over `bash`?

Comment (drozdowsky) [reply .pre-commit-config.yaml]: Hi,
sh is used more widely, in old days /bin/sh was linked to /bin/bash but nowadays some distros/OSes do not ship with /bin/bash and /bin/sh can link e.g. to /bin/dash.
That is the case for me, FreeBSD does not ship with bash by default.

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - JWT payload now contains user global id - #5039 by @salwator
 - Made middleware path resolving lazy and refactored middleware names - #5041 by @NyanKiyoshi
 - Generate slug in attribute value save - #5055 by @fowczarek
+- Change bash to sh - #5064 by @drozdowsky
```
Comment (NyanKiyoshi) [inline CHANGELOG.md]: I don't think this is needed to be put in the changelog

Comment (drozdowsky) [reply CHANGELOG.md]: Ok, removed

**Comment by maarcingebala** [comment]
Thanks for your contribution!

**Pull request merged**